src/platforms/esp32/test/main/test_main.c

```diff
@@ -489,13 +494,17 @@ TEST_CASE("test_ssl", "[test_run]")
     TEST_ASSERT(ret_value == OK_ATOM);
 }
 
+// Works C3 on local runs, but fails GH actions
```

Comment:
just to understand, how it fails?

Reply:
https://github.com/atomvm/AtomVM/actions/runs/8205149637/job/22441379197#step:13:4571

2024-03-08 15:12:06 Running test_rtc_slow...
2024-03-08 15:12:06 I (30942) network_driver: Initialized network interface
2024-03-08 15:12:06 I (30942) network_driver: Created default event loop
2024-03-08 15:12:06 I (30942) AtomVM: Running start/0 from test_rtc_slow.beam...
2024-03-08 15:12:06 
2024-03-08 15:12:06 CRASH 
2024-03-08 15:12:06 ======
2024-03-08 15:12:06 pid: <0.1.0>
2024-03-08 15:12:06 
2024-03-08 15:12:06 
2024-03-08 15:12:06 Stacktrace:
2024-03-08 15:12:06 undefined
2024-03-08 15:12:06 
2024-03-08 15:12:06 
2024-03-08 15:12:06 cp: #CP<module: 0, label: 27, offset: 0>
2024-03-08 15:12:06 
2024-03-08 15:12:06 
2024-03-08 15:12:06 x[0]: error
2024-03-08 15:12:06 x[1]: {badmatch,<<0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,43>>}
2024-03-08 15:12:06 x[2]: error
2024-03-08 15:12:06 
2024-03-08 15:12:06 
2024-03-08 15:12:06 Stack 
2024-03-08 15:12:06 ------
2024-03-08 15:12:06 
2024-03-08 15:12:06 
2024-03-08 15:12:06 
2024-03-08 15:12:06 []
2024-03-08 15:12:06 []
2024-03-08 15:12:06 #CP<module: 0, label: 27, offset: 0>
2024-03-08 15:12:06 
2024-03-08 15:12:06 
2024-03-08 15:12:06 
2024-03-08 15:12:06 Mailbox
2024-03-08 15:12:06 
2024-03-08 15:12:06 --------
2024-03-08 15:12:06 
2024-03-08 15:12:06 
2024-03-08 15:12:06 
2024-03-08 15:12:06 Monitors
2024-03-08 15:12:06 --------
2024-03-08 15:12:06 
2024-03-08 15:12:06 
2024-03-08 15:12:06 
2024-03-08 15:12:06 **End Of Crash Report**
2024-03-08 15:12:06 AtomVM finished with return value: error
2024-03-08 15:12:06 ./main/test_main.c:500:test_rtc_slow:FAIL:Function [test_run].  Expression Evaluated To FALSE

Reply:
test_rtc_slow - succeeds local qemu macos - fails "act"/docker arm64 - fails gh action